tensorflow/python/kernel_tests/bias_op_test.py

```diff
@@ -128,7 +128,13 @@ def _testGradient(self, np_input, bias, dtype, data_format, use_gpu):
           input_tensor, np_input.shape, output_tensor, np_input.shape)
       bias_jacob_t, bias_jacob_n = tf.test.compute_gradient(
           bias_tensor, bias.shape, output_tensor, np_input.shape)
-
+         
+      # Test gradient of BiasAddGrad
+      bias_add_grad = tf.gradients(tf.nn.l2_loss(output_tensor),
+                                   bias_tensor)[0]
+      grad_jacob_t, grad_jacob_n = tf.test.compute_gradient(
+          output_tensor, np_input.shape, bias_add_grad, bias.shape)
```

Comment:
To clarify, right now the data_format of the bias op is being compared to a Python string literal. Looking at [another similar code snippet](https://github.com/tensorflow/tensorflow/blob/450e9386e0cc4b81b49adb9184859411f43ef259/tensorflow/python/framework/common_shapes.py#L185) suggests that should explicitly make it an ASCII string. I'll fix that shortly.